raindrops/script.js

```diff
@@ -0,0 +1,584 @@
+        }
+    }
+    if (lastOperator == '*' && !manualSetOperand) {
+        lastSecondOperand = Math.round(lastSecondOperand / 2);    
+    }
+    return lastSecondOperand;
+}
+
+function pressCalcBtn(value) {
+    if (value === 'Clear') {
+        displayOutput.value = '';
+    } else if (value === 'Delete') {
+        displayOutput.value = displayOutput.value.slice(0, -1);
+    } else if (value === 'Enter') {
+        checkAnswer(displayOutput.value);
+        displayOutput.value = '';
+    } else {
+        if (+displayOutput.value > 999) {
+            displayOutput.value = displayOutput.value;
+        } else if (displayOutput.value === '' && value === '0') {
+            displayOutput.value = '';
+        } else {
+            displayOutput.value += value;
+        }
+    }
+};
+
+function pressKeyboard(e) {
+    if (e.code.includes('Digit') || e.code.includes('Numpad') && (e.keyCode >= 96 && e.keyCode <= 105)) {
+        if (+displayOutput.value > 999) {
+            displayOutput.value = displayOutput.value;
+        } else if (displayOutput.value === '' && e.code.includes('0')) {
+            displayOutput.value = '';
+        } else {
+            displayOutput.value = displayOutput.value + e.key;
+        }
+    };
+    if (e.code.includes('Enter')) {
+        checkAnswer(displayOutput.value);
+        displayOutput.value = '';
+    }
+    if (e.code.includes('Backspace')) {
+        displayOutput.value = displayOutput.value.slice(0, -1);
+    }
+}
+
+function checkAnswer() {
+    if (rainDropsArr[0].result === Number(displayOutput.value)) {
+        playCorrectAnswerSound();
+        getGameScore('plus');
+        ++counterRightAnswers;
+        rainDropsArr[0].wasDecided = true;
+        rainDropsArr[0].element.style.transform = 'scale(0)';
+        rainDropsArr[0].element.style.transition = 'transform 1s';
+        if (rainDropsArr[0].rainDropExtra === true) {
+            rainDropsArr[0].element.style.boxShadow = '0 0 5px 10px rgba(187, 255, 0, 0.8)';
+            rainDropsArr.shift();
+            rainDropsArr.forEach(e => {
+                e.element.remove();
+                e.wasDecided = true;
+            });
+            rainDropsArr.length = 0;
+            playZone.style.height = '';
+            waterZone.style.height = '';
+        } else {
+            rainDropsArr[0].element.style.boxShadow = '0 0 5px 10px rgba(113, 214, 230, 0.8)';
+            rainDropsArr.shift();
+        }
+    } else {
+        playWrongAnswerSound();
+        rainDropsArr[0].element.style.boxShadow = '0 0 5px 10px rgba(255, 69, 0, 0.8)';
+        setTimeout(() => {
+            rainDropsArr[0].element.style.boxShadow = '';
+        }, 100);
+        ++counterWrongAnswers;
+    }
+}
+
+function toggleSound() {
+    waveSound.autoplay = false;
+    waveSound.volume = 0.4;
+    if (waveSound.paused && waterZone.style.height !== maxWaterLevel) {
+        waveSound.play();
+        audioIcon.innerHTML = '&#128265;';
```

Comment:
'&#128265;' - also magic value

It's better to create understanding constant for it.
Something like it: 
```
const volumeIcon = '&#128265;'
```